content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -250,14 +257,20 @@
 	private final String PROP_ACCEPTABLE_FILE_TYPES = "turnitin.acceptable.file.types";
 
 	private final String KEY_FILE_TYPE_PREFIX = "file.type";
+	
+	private String autoExcludeSelfMatchingScope;
 
 	public void init() {
 		EULA_CACHE = memoryService.createCache("org.sakaiproject.contentreview.turnitin.oc.ContentReviewServiceTurnitinOC.LATEST_EULA_CACHE", new SimpleConfiguration<>(10000, 24 * 60 * 60, -1));
 		// Retrieve Service URL and API key
 		serviceUrl = serverConfigurationService.getString("turnitin.oc.serviceUrl", "");
 		apiKey = serverConfigurationService.getString("turnitin.oc.apiKey", "");
 		// Retrieve Sakai Version if null set default 		
-		sakaiVersion = Optional.ofNullable(serverConfigurationService.getString("version.sakai", "")).orElse("UNKNOWN");		
+		sakaiVersion = Optional.ofNullable(serverConfigurationService.getString("version.sakai", "")).orElse("UNKNOWN");
```

Comment:
Does passing the UNKOWN as a default to the getString call make sense?

Reply:
I didn't modify that in this PR, but looking at this code, it prob can never be "UNKNOWN" since the default is `""`. It prob makes sense to make that change.

Reply:
updated